nucypher/cli/commands/stake.py

```diff
@@ -431,9 +432,10 @@ def create(general_config, transacting_staker_options, config_file, force, value
 
     if not lock_periods:
         min_locktime = STAKEHOLDER.economics.minimum_locked_periods
-        max_locktime = STAKEHOLDER.economics.maximum_rewarded_periods
+        default_locktime = STAKEHOLDER.economics.maximum_rewarded_periods
+        max_locktime = SOLIDITY_MAX_UINT16 - STAKEHOLDER.staking_agent.get_current_period()
         prompt = f"Enter stake duration ({min_locktime} - {max_locktime})"
-        lock_periods = click.prompt(prompt, type=stake_duration_range, default=max_locktime)
+        lock_periods = click.prompt(prompt, type=stake_duration_range, default=default_locktime)
```

Comment:
Is it ever possible for `default_locktime` to be greater than `max_locktime`? Feels like a very small possibility when current period is large enough (?), but I can't quite wrap my head around it...

Reply:
You're right, after 06/06/2148 it will become a problem :) I guess current code will face with other problems closer to this date so I prefer to keep it for a few decades :)

Reply:
Hahaha touché!